.go-version

```diff
@@ -1 +1 @@
-1.17
+1.18
```

Comment:
I guess this should not cause any issues, but let's wait for the whole e2e-testing runs in the CI